.github/workflows/ci-run-test.yml

```diff
@@ -140,6 +140,10 @@ jobs:
           } >> "$GITHUB_ENV"
           # For debugging:
           ps aux
+      - name: Install tsc on windows and macos
+        if: ${{ runner.os == 'windows' || runner.os == 'macOS' }}
+        run: |
+          npm install typescript -g
```

Comment:
? Is this needed for the new integration test? Doesn't its npm package cover it

Reply:
Yes. Windows somehow doesn't seem to be picking up `tsc` properly otherwise: https://github.com/pulumi/pulumi/actions/runs/13964609930/job/39092245402?pr=18951.  I remember the same thing being an issue on MacOS, from https://github.com/pulumi/pulumi/pull/18055.